service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToStartTest.java

```diff
@@ -92,6 +96,19 @@ void shouldNotAddAmountToPayFieldWhenFeatureToggleIsFalse() {
         assertThat(response.getData()).doesNotContainKey("amountToPay");
     }
 
+    @Test
+    void shouldNotDisplayAmountToPayFieldWhenErrorIsThrown() {
+        given(ldClient.boolVariation(eq("FNP"), any(), anyBoolean())).willReturn(true);
+        given(feeService.getFeeAmountForOrders(any())).willThrow(new FeeRegisterException(300, "duplicate", null));
```

Comment:
Super piiiicky, I don't like status error 300. 
300 Multiple Choices
Indicates multiple options for the resource from which the client may choose (via agent-driven content negotiation). For example, this code could be used to present multiple video format options, to list files with different filename extensions, or to suggest word-sense disambiguation

I wouold suggest 409 it this instance as this would exactly mimic your "duplicate". 

Sorry but I am pedantic with the error codes!

Reply:
Chose 300 as that is the error I have most experience receiving from the fee-register but yeah can change to 409

Reply:
Oh did they return this as 300?

Reply:
yes